src/contents/all/components/app/LoginForm.scss

```diff
@@ -0,0 +1,228 @@
+
+  .ant-form-item.borderTop {
+    border-top: 1px solid #2a2a2a;
+    padding-top: 20px;
+  }
+
+  .ant-form-item.name {
+
+    .ant-input-affix-wrapper {
+      border: none !important;
+      width: calc(50% - 7px);
+      display: inline-block;
+      input.ant-input {
+        width: 100%;
+        float: none;
+      }
+
+      .ant-input-prefix {
+        color: #aaa;
+      }
+    }
+    input.ant-input {
+      width: calc(50% - 7px);
+      float: right;
+    }
+  }
+
+  .stickyBottom {
+    position: absolute;
+    bottom: 0;
+    width: 100%;
+    border-top: 1px solid #2a2a2a;
+    padding: 10px 16px;
+    text-align: right;
+    left: 0;
+    z-index: 9;
+    background: #1a1a1a;
+
+    .ant-btn {
+      float: right;
+      width: auto;
+    }
+
+    .link {
+      margin: 7px 12px 0 10px;
+      float: left;
+      transition: all 0.3s ease;
+      font-size: 14px;
+      text-transform: uppercase;
+      cursor: pointer;
+      &:hover {
+        color: #c7ac75;
+      }
+    }
+  }
+
+  .ant-form-item:last-child {
+    margin-bottom: 10px;
+  }
+  .anticon-check {
+    color: #aaa !important;
+  }
+  .ant-btn {
+    display: block;
+    width: 100%;
+    font-weight: 400;
+    font-size: 13px;
+    .anticon {
+      color: #c7ac75 !important;
+    }
+  }
+
+  .textAreaGroup {
+    position: relative;
+    .iconCol {
+      position: absolute;
+      left: 11px;
+      top: 10px;
+    }
+
+    .textAreaCol {
+      padding-left: 27px;
+      -webkit-transition: ease .2s all;
+    }
+  }
+
+  label {
+    font-size: 16px;
+    color: #aaa;
+    text-transform: uppercase;
+    font-weight: 300;
+    margin-left: 12px;
+    font-family: "Montserrat", sans-serif;
+
+    .anticon.anticon-solution {
+      vertical-align: middle;
+      margin-right: 5px;
+    }
+  }
+
+  textarea.ant-input, input.ant-input {
+    border: none;
+    font-family: 'Montserrat', sans-serif !important;;
+    font-weight: 300;
+    resize: none;
+
+    &:active, &:focus, &:hover {
+      border: none;
+      outline: 0;
+      box-shadow: none;
+    }
+  }
+
+  .has-error .ant-form-explain {
+    font-family: 'Montserrat', sans-serif;
+    font-weight: 300;
+    padding-left: 12px;
+  }
+  .has-error .ant-input-affix-wrapper .ant-input:focus {
+    border: none !important;
+    box-shadow: none;
+  }
+
+  .ant-alert {
+    border: none;
+    width: 100%;
+    position: absolute;
+    bottom: auto;
+    left: 0;
+    top: -50px;
+    text-align: center;
+    font-family: 'Montserrat', sans-serif;
+    animation: slidein 0.4s;
+    -webkit-animation-fill-mode: forwards;
+    animation-fill-mode: forwards;
+  }
+}
+
+.formFooter {
+  -webkit-animation: fadein 0.7s;
+  animation: fadein 0.7s;
+  opacity: 0;
+  -webkit-animation-delay: 0.6s;
+  animation-delay: 0.6s;
+  -webkit-animation-fill-mode: forwards;
+  animation-fill-mode: forwards;
+  text-align: center;
+  margin: 30px;
+  font-size: 13px;
+  color: #aaa;
+  font-family: "Montserrat", sans-serif;
```

Comment:
all of these fonts are not rendering. getting a console error Mixed Content: `The page at 'https://divisare.com/projects/295412-lan-architecture-julien-lanoo-urban-renovation-lormont' was loaded over HTTPS, but requested an insecure stylesheet 'http://fonts.googleapis.com/css?family=Montserrat:300,400'. This request has been blocked; the content must be served over HTTPS.`